### PR TITLE
Adds syndicate cyborgs. Adds Assault cyborgs to nukie uplink

### PR DESCRIPTION
**Pull request by Naive817**
## About the PR

Add three models of syndicate cyborg chassises to the game: Assault, Medical and Saboteur.
They have no access by default. They have syndicate laws by default. They have noslip and MovementAlwaysTouching components as for now (like space carps) because cyborg magboots and jetpacks aren't in the code yet.
Add several syndicate cyborg modules to the game.
-Operative module: contains a crowbar, a cobra, an Emag and a pinpointer
-L6C ROW module: basically a L6 with 100 rounds, cannot be reloaded, accuracy is same with a wielded L6
-E sword module: an esword
Implements the assault cyborg to nukie uplink for 65 TC, comes with an Operative module, a L6C module, an esword module , a positronic brain & a hyper capacity powercell. Has 200 HP.

## Why / Balance

This is to add more options for nukies when they get the TC boost from war declaration. They can use a hefty part of their TC to buy a reinforcement cyborg, which is more well armed than regular reinforcements.
The 65 TC is actually less than reinforcement (16 TC), L6 (40 TC), e sword (8 TC), Emag (8 TC), Bloodred hardsuit (8 TC) combined (80 TC), but cyborgs have their own weakness: being not able to pick up things so they are in general less versitile, also can't as easily be healed. And their battery might run out.
Might need some rounds to test it's balance

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
imagew

https://github.com/space-wizards/space-station-14/assets/31364560/c68d9f8d-12b5-499a-b2ba-1470ed0b9bb8

## Breaking changes

**Changelog**

:cl:
- add: Syndicate has fielded cyborgs for testing. They are loaded with syndicate laws and have no access.
- add: Nuclear operatives can now call in an well-armed assault cyborg for a hefty price (65 TC). Comes with an Operative module complete with Emag and pinpointer, a L6C ROW module, an Esword module, a positronic brain & a hyper capacity powercell

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml [hunk 1]
```diff
@@ -77,3 +77,35 @@
...
+      selectedMode: FullAuto
+      availableModes:
+        - FullAuto
+      soundGunshot:
+        path: /Audio/Weapons/Guns/Gunshots/lmg.ogg
+      soundEmpty:
+        path: /Audio/Weapons/Guns/Empty/lmg_empty.ogg
+    - type: BallisticAmmoProvider
+      proto: CartridgeLightRifle
+      capacity: 100
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml:111]: Shouldn't this consume battery to use instead, seems to make a lot more sense.

**Comment by CaptainSqrBeard** [comment]
You have two more borgs into PR but they are not used. Is it intentional?
Also can borg emag itself?

**Comment by Naive817** [comment]
> You have two more borgs into PR but they are not used. Is it intentional? Also can borg emag itself?

I only ported the the other two borgs' sprite over, haven't got any good way to implement them.
The syndicate medical cyborg would be too weak in the current state of the game and the saboteur cyborg would need chameleon function.
I tried and the Borg can't emag itself.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml (hunk 1 above)
Comment (Naive817) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml:111]: Yes they should use power for bullet but that would require changing borg code, which I haven't got around to do (and I doubt whether I can code it)

**Comment by Holinka4ever** [comment]
What about to give syn borgs a nuclear gripper? Aka a hand that designed to hold nuclear disk.
To like kill cap and then retreat with disk thing...
Also, they should have a crowbar though, if lights in station would go off.

**Comment by FellowsinSpess** [comment]
I feel like the Cobra can be Swapped for the energy sword in the Operative module. Feels odd having one module for just an Esword.

**Comment by Naive817** [comment]
> What about to give syn borgs a nuclear gripper? Aka a hand that designed to hold nuclear disk. 

The cyborg code doesn't support something like this at the moment.... Might need to be another PR when I figure it out

>To like kill cap and then retreat with disk thing... Also, they should have a crowbar though, if lights in station would go off.
 
The operative module does have a crowbar(
 
>I feel like the Cobra can be Swapped for the energy sword in the Operative module. Feels odd having one module for just an Esword.

Operative Module in my opinion should be the base module for all three kinds of cyborgs, like nukie's starting kit, so I intentionally put a weak weapon in it. Maybe Viper would be better?
But swapping between modules as cyborg can be annonying, so I added pinpointer to each module as for now.

**Comment by deltanedas** [comment]
ninja syndiborg conversion real

**Comment by JoeHammad1844** [comment]
> I only ported the the other two borgs' sprite over, haven't got any good way to implement them. The syndicate medical cyborg would be too weak in the current state of the game (no surgery)

also no medibeam

**Comment by JoeHammad1844** [comment]
still nothing after a month?

**Comment by metalgearsloth** [comment]
> still nothing after a month?

what are you expecting after a month?

**Comment by JoeHammad1844** [comment]
well, more than nothing

**Comment by metalgearsloth** [comment]
> well, more than nothing

Random PRs are not the place to complain about merge times as this is a volunteer project.

**Comment by JoeHammad1844** [comment]
im complaining that specifically this PR has been ignored

**Comment by ps3moira** [comment]
don't let this PR die!!

**Review by Emisse — APPROVED** [review]

**Comment by Emisse** [comment]
fix conflicts and looks good

**Comment by K-Dynamic** [comment]
Aggghghbgh final stretch

**Comment by Hmeister-fake** [comment]
Just a week away...

**Comment by Naive817** [comment]
> fix conflicts and looks good

sorry for ghosting this PR so long, will come back after 1/10
however cyborgs' equipment slots still act like hands

**Comment by Ilya246** [comment]
i would say 65 TC is way too expensive considering the L6 can't be reloaded (it normally at least comes with 2 mags) and this thing doesn't even have hands or a full set of tools/access, it has to wait for its emag to recharge to get anywhere, even maintenance
i would suggest to give it the capability to scan IDs on touch like agent ID

**Comment by K-Dynamic** [comment]
On the other hand, it's very dangerous to fight against as the borg has no slowdown and can be easily welded back to full health

I'm worried about:
- how it'll interact with EMPs from teammates

- if it should be able to strip and get the nuke disk from cap, as well as inserting nuke disk

- whether noslip component is redundant as all borgs have noslips, same with the touching wall component

- if it counts as part of nukie threat for endscreen and automatic evac call when all nukies die

**Comment by Ilya246** [comment]
> whether noslip component is redundant as all borgs have noslips, same with the touching wall component

the AlwaysTouching component lets it move in space like spaec mobs

**Comment by K-Dynamic** [comment]
> > whether noslip component is redundant as all borgs have noslips, same with the touching wall component
> 
> the AlwaysTouching component lets it move in space like space mobs

My bad, I was referring to the time borgs couldn't move near walls in space, this is a redundant point then

**Comment by TheShuEd** [comment]
how's the development going?

**Comment by TheShuEd** [comment]
Syndie borg is real.

**Pull request merged**